TODO/machine-learning-for-android-developers-with-the-mobile-vision-api-part-1-face-detection.md

```diff
@@ -71,15 +71,15 @@ _Note_: If you were building a game called _!(Recognize Me)_ where you need to
     canvas.drawBitmap(defaultBitmap, 0, 0, null);
 ```
 
-*   Finally, we get to point where we use the _FaceDectector_ API. Tracking is disabled because we are using a static image. It should be enabled for videos.
+* 最后，让我们言归正传，说说看怎么使用 _FaceDectector_ API 。因为我们在使用一个静态的图片，所以追踪功能被禁止了。它会在视频上被启用。
 
 ```
     FaceDetector faceDetector = new FaceDetector.Builder(this)
             .setTrackingEnabled(false)
             .setLandmarkType(FaceDetector.ALL_LANDMARKS)
             .build();
 ```
-*   Check if the face detector is operational already. There’s a possibility that it won’t work the first time because a library needs to be downloaded to the device and it might not have been completed in time when you need to use it.
+* 检查是否人脸侦测正常运作了。有可能第一次它不能正常工作，因为有一个依赖库需要被下载到设备上，并且当你需要使用它的时候还没有完全下载好。
```

Comment:
最后一句 "a library needs to be downloaded to the device **and** it might not have been completed in time when you need to use it" 中的 "and" 表示的是动作的连续，不要硬生生翻译成“并且”。 这样翻译“因为一个依赖库需要备下载到设备上，而当你需要使用它的时候 可能 还没有下载好。"